test/tests/dirackernels/vectorPostprocessor_point_source/3d_vpp.i

```diff
@@ -0,0 +1,67 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 10
+  ny = 10
+  nz = 10
+[]
+
+[Variables]
+  active = 'u'
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+[]
+
+[DiracKernels]
+  [./point_source]
+    type = VectorPostprocessorPointSource
+    variable = u
+    vector_postprocessor = csv_reader
+    x_coord_name = x3
+    y_coord_name = y3
+    z_coord_name = z3
+    value_name = value3
+  [../]
+[]
+
+[VectorPostprocessors]
+  [./csv_reader]
+    type = CSVReader
+    csv_file = point_value_file.csv
+    execute_on = initial
+  [../]
+[]
+
+[BCs]
+  [./left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  [../]
+  [./right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  [../]
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = 'PJFNK'
+[]
+
+[Outputs]
+  file_base = 3d_out
```

Comment:
let's remove all those file_base params, they make it harder to see which output corresponds to which input

Reply:
This is so the output file can be diffed with the gold file from constant point source.

Reply:
my preference would be a symlink in the gold directory, but I think we may not have an "official" stance on that

Reply:
I changed these to not use the file_base and to use a symlink with a different name than the file being linked.